vnext/Microsoft.ReactNative/Fabric/Composition/TextInput/WindowsTextInputComponentView.cpp

```diff
@@ -1123,19 +1121,11 @@ void WindowsTextInputComponentView::updateState(
     m_comingFromState = true;
     // Only handle single/empty fragments right now -- ignore the other fragments
 
-    LRESULT res;
```

Comment:
Removed this code because UpdateText calls the same messages. If these extra calls are needed, I can add them back!